src/algorithms/calorimetry/CalorimeterClusterRecoCoG.cc

```diff
@@ -141,8 +150,9 @@ void CalorimeterClusterRecoCoG::AlgorithmProcess() {
         edm4eic::MCRecoClusterParticleAssociation* cassoc = new edm4eic::MCRecoClusterParticleAssociation(*clusterassoc);
         m_outputAssociations.push_back(cassoc);
       } else {
-        if (false) {
-          LOG_INFO(default_cout_logger) << "No mcHitCollection was provided, so no truth association will be performed." << LOG_END;
+        if (m_logger->level() == spdlog::level::debug) {
```

Comment:
It is just a 1/3 of changes... *<=*

Reply:
fixed (neglected in my find and replace) ...in incoming push